eqcorrscan/tests/match_filter_test.py

```diff
@@ -1242,11 +1242,16 @@ def test_party_rethreshold(self):
             det.id = str(i)
             party[0].detections.append(det)
         self.assertEqual(len(party), 204)
-        party.rethreshold(new_threshold=9)
-        for family in party:
+        party1 = party.copy().rethreshold(new_threshold=9)
+        for family in party1:
             for d in family:
                 self.assertEqual(d.threshold_input, 9.0)
                 self.assertGreaterEqual(d.detect_val, d.threshold)
+        party2 = party.copy().rethreshold(new_threshold=9, abs_values=True)
+        for family in party2:
+            for d in family:
+                self.assertEqual(d.threshold_input, 9.0)
+                self.assertGreaterEqual(abs(d.detect_val), d.threshold)
```

Comment:
Should check that this actually has some large negative detect_vals that are retained.

Reply:
It looks like there are no negative values retained. I'm going to add a check here, make sure the test fails in the CI, then update the test.